dapps/assets/src/components/digitaltwins/AddDigitalTwin.ts

```diff
@@ -17,41 +17,123 @@
   the following URL: https://evan.network/license/
 */
 
-// vue imports
 import Component, { mixins } from 'vue-class-component';
-
-// evan.network imports
 import { EvanComponent } from '@evan.network/ui-vue-core';
+import { UIContainerFile } from '@evan.network/ui';
+import { ContainerPlugin } from '@evan.network/api-blockchain-core';
+
+// load twin templates
+import bicycleTwin from './templates/bicycle.json';
+import carTwin from './templates/car.json';
+
+// TODO: get from common interfaces
+interface DigitalTwinTemplate {
+  description: {
+    description?: string;
+    i18n?: {
+      [language: string]: {
+        description?: string;
+        name: string;
+      }
+    };
+    name: string;
+  };
+  plugins: { [pluginName: string]: ContainerPlugin };
+}
 
 @Component
-export default class AddDigitalTwinComponent extends mixins(EvanComponent) {
-  name: string = null;
-  desc: string = null;
-  template = null;
+class AddDigitalTwinComponent extends mixins(EvanComponent) {
+  description: string = null;
   image = null;
+  name: string = null;
+  selectedTemplate = 'car';
+  template = <DigitalTwinTemplate>carTwin;
   presetTemplates = [
```

Comment:
We should use an object that contains all template definitions and build a dynamic presetTemplates array, so that we are prepared for the future.

e.g. (warning, not tested):

```ts
this.twins = {  bicycleTwin, carTwin };
this.selectedTemplate = Object.keys(this.twin)[0];
this.presetTemplates = Object.keys(twins).map(twinKey => {
  return {
    value: twinKey,
    label: this.getLocalizedTemplateEntry(this.twins.i18n, 'name'),
  }
});
```

As a result of this, `handleTemplateSelectChange` could be dropped and the template could be accessed with `twins[selectedTemplate]. Also `handleFileUpload` for new templates could be added into the `twins` object.